sql-plugin/src/main/scala/org/apache/spark/sql/rapids/datetimeExpressions.scala

```diff
@@ -1044,29 +1044,34 @@ class FromUTCTimestampExprMeta(
     rule: DataFromReplacementRule)
   extends BinaryExprMeta[FromUTCTimestamp](expr, conf, parent, rule) {
 
+  private[this] var timezoneId: ZoneId = null
+
+  lazy val supportedZoneStrList = Seq("UTC", "America/Los_Angeles", "Asia/Shanghai")
```

Comment:
Phase 1, we only support non-repeat time zone.
America/Los_Angeles will not work.